src/protocol/thread/bounds.ts

```diff
@@ -22,19 +22,20 @@ export class NodeBoundsFront {
   }
 
   getBoxQuads(box: any) {
```

Comment:
I think this was added so that this method has the same signature as the one in `NodeFront`. It's not made explicit anywhere but `NodeFront` and `NodeBoundsFront` have a common interface because both can be used by the highlighter (that is also the reason for the `isNodeBoundsFront()` method).